tap-client/src/components/home/HomeComponent.css

```diff
@@ -0,0 +1,49 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: Arial;
+    padding: 20px;
+    background-color: #dddddd;
+}
+
+.Header {
```

Comment:
Be consistent with naming conventions, use camel case

Reply:
Corrected to the best of my knowledge.